src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorDataSourceRetrieverIntegrationTest.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.hellbender.tools.funcotator;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+/**
+ * Class to test the {@link FuncotatorDataSourceRetriever}.
+ * Created by jonn on 8/24/18.
+ */
+public class FuncotatorDataSourceRetrieverIntegrationTest extends CommandLineProgramTest {
+
+    //==================================================================================================================
+    // Private Static Members:
+
+    private static final boolean doDebugTests = false;
+
+    //==================================================================================================================
+    // Private Members:
+
+    //==================================================================================================================
+    // Helper Methods:
+
+    //==================================================================================================================
+    // Data Providers:
+
+    //==================================================================================================================
+    // Tests:
+
+    @Test(enabled = doDebugTests,
+            groups = {"funcotatorValidation", "bucket"}
+    )
+    void testGermlineDownloadWithValidation() {
```

Comment:
Add test for somatic download as well.

Reply:
Will do.

@droazen In practice will these tests ever be run?  Frankly they take so long that I doubt anyone will enable them.